pkg/kine/logstructured/sqllog/sql.go

```diff
@@ -397,10 +399,15 @@ func (s *SQLLog) poll(result chan interface{}, pollStart int64) {
 			}
 		}
 		waitForMore = true
+		watchCtx, cancel := context.WithTimeout(s.ctx, s.d.GetWatchQueryTimeout())
+		defer cancel()
 
-		rows, err := s.d.After(s.ctx, last, 500)
+		rows, err := s.d.After(watchCtx, last, 500)
 		if err != nil {
 			logrus.Errorf("fail to list latest changes: %v", err)
```

Comment:
nit: remove this. This should only be printed if error != context.DeadlineExceeded